packages/language/src/preprocessor/instruction-interpreter.ts

```diff
@@ -1385,13 +1383,15 @@ function parseAndEvaluateProcedure(
       context,
     );
   }
-
   const localContext = createLocalContext(context, procedure);
   let value = runProcedure(procedure, evaluatedArgs, localContext);
-  if (isScalarValue(value) && suffix) {
+  if (isScalarValue(value) && immediateFollow && tokens[index + advance]) {
```

Comment:
Move `const nextToken = tokens[index + advance];` before the `if`. Then you do not have to get it twice.